### DISCUSSION TITLE
ADB ISSUE

### DISCUSSION THREAD

**Discussion by ASSIMILATORS** [termux/termux-app#2471] [Q&A]
Kindly advise as i am not able to use adb on termux attached is the error log

* daemon not running; starting now at tcp:5037
ADB server didn't ACK
Full server startup log: /data/data/com.termux/files/usr/tmp/adb.12224.log
Server had pid: 26553
--- adb starting (pid 26553) ---
adb I 12-14 07:21:59 26553 26553 main.cpp:63] Android Debug Bridge version 1.0.41
adb I 12-14 07:21:59 26553 26553 main.cpp:63] Version 31.0.3-android-tools
adb I 12-14 07:21:59 26553 26553 main.cpp:63] Installed as /data/data/com.termux/files/usr/bin/adb
adb I 12-14 07:21:59 26553 26553 main.cpp:63]
adb F 12-14 07:22:00 26553 26553 main.cpp:155] could not install *smartsocket* listener: Address already in use

* failed to start daemon
adb: failed to check server version: cannot connect to daemon

**Comment by agnostic-apollo** [comment]
Known issue https://github.com/termux/termux-packages/issues/7946.

**Reply by ASSIMILATORS** [reply]
> Known issue [termux/termux-packages#7946](https://github.com/termux/termux-packages/issues/7946).

Thanks for reply but I belive my problem started only after I installed https://github.com/MuntashirAkon/libadb-android yesterday. I have also asked the app developer about it waiting for its reply.

**Reply by ASSIMILATORS** [reply]
My problem got resolved after installing adb from following link

https://github.com/rendiix/termux-adb-fastboot
